app/src/main/res/values/strings.xml

```diff
@@ -82,11 +84,16 @@
         <item quantity="one">%d day</item>
         <item quantity="other">%d days</item>
     </plurals>
+    <string name="textFieldOptional">(Optional)</string>
+    <string name="transferMessagePlaceholder">A message to pass on?</string>
```

Comment:
Is "pass on" the correct verb? Because I feel like it has a strange connotation for this context. Maybe something along the lines of "transmit" or similar other words would fit better?

Reply:
We search in dictionnaries, turned out pass on is in fact the correct verb